### PR TITLE
Add new static-delta command

### PR DESCRIPTION
**Pull request by doanac**
Signed-off-by: Andy Doan <andy@foundries.io>

### CODE REVIEW

**Comment by doanac** [comment]
@mike-sul - you probably want to look at this also

File: subcommands/targets/deltas.go [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+)
+
+func init() {
+	var deltas = &cobra.Command{
+		Use:   "static-deltas <target-version> [<from-version>...]",
+		Short: "Generate static deltas to the given target version to make OTAs faster",
+		Run:   doDeltas,
+		Args:  cobra.MinimumNArgs(1),
+		Long: `In many cases OTAs will have many ostree changes. These OTAs can be 
+sped up significantly by generating OSTree static deltas. Static deltas
```
Comment (mike-sul) [inline subcommands/targets/deltas.go:27]: `sped` --> `speed`

File: subcommands/targets/deltas.go [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+	dryRun bool
+)
+
+func init() {
+	var deltas = &cobra.Command{
+		Use:   "static-deltas <target-version> [<from-version>...]",
+		Short: "Generate static deltas to the given target version to make OTAs faster",
+		Run:   doDeltas,
+		Args:  cobra.MinimumNArgs(1),
+		Long: `In many cases OTAs will have many ostree changes. These OTAs can be
```
Comment (mike-sul) [inline subcommands/targets/deltas.go:26]: I am not sure about it, just think that `OTAs` should be `OTA updates` since OTA stands just for Over-The-Air?

**Review by mike-sul — APPROVED** [review]
Just minor comments. I actually tried it by following the guide https://github.com/foundriesio/docs/pull/213/files and it worked for me, device/aklite pulled static deltas.

**Comment by mike-sul** [comment]
Maybe it makes sense to check whether `from-version` is actually lower than `target-version` to forbid doing something like this `fioctl targets static-deltas 803 806` which doesn't make much sense.

File: subcommands/targets/deltas.go (hunk 2 above)
Comment (doanac) [reply subcommands/targets/deltas.go:26]: i fear we probably say "OTAs" or just OTA in many places now. After a big of googling i'll say Android is probably the best to follow:

 https://source.android.com/devices/tech/ota

they say "OTA updates". I'll fix this to not make things worse, but there's probably a lot of stuff docs/blogs/code that repeats this.

File: subcommands/targets/deltas.go (hunk 1 above)
Comment (doanac) [reply subcommands/targets/deltas.go:27]: no this is "sped up" as in made faster. maybe i should say "These updates can be downloaded faster by generating .."

Comment (doanac) [reply subcommands/targets/deltas.go]: redid the wording - https://github.com/foundriesio/fioctl/pull/99/commits/abb98ca62cfc1ce83520149e69977c0ff7656ee8

File: subcommands/targets/deltas.go (hunk 2 above)
Comment (doanac) [reply subcommands/targets/deltas.go]: https://github.com/foundriesio/fioctl/pull/99/commits/abb98ca62cfc1ce83520149e69977c0ff7656ee8

**Comment by doanac** [comment]
> Maybe it makes sense to check whether from-version is actually lower than target-version

Good idea: https://github.com/foundriesio/fioctl/pull/99/commits/4bf5fb03579e7f649216e23dcb13b13acc01b57e

**Comment by doanac** [comment]
squash push

**Comment by doanac** [comment]
rebase on master

**Pull request merged**